controllers/k8ssandra/cleanup.go

```diff
@@ -85,6 +88,20 @@ func (r *K8ssandraClusterReconciler) checkDeletion(ctx context.Context, kc *api.
 			}
 		}
 
+		// Delete the Stargate generated config map
+		sgConfigMap := corev1.ConfigMap{}
```

Comment:
There should be no need to delete the ConfigMap if you set an owner reference on it. The delete will cascade when the Stargate object is deleted.

Reply:
done 👍